cognite/pygen/utils/helper.py

```diff
@@ -21,3 +21,35 @@ def chdir(new_dir: Path) -> Iterator[None]:
 
     finally:
         os.chdir(current_working_dir)
+
+
+def _get_literal_str(text: str) -> str | None:
```

Comment:
Can you add a few unit tests for this function?

Reply:
In addition to the doctest?

Reply:
🔝 disregard my comment. I was not aware you are using doctest ;)